lib/utils.py

```diff
@@ -4,38 +4,42 @@
 import sys
 import gzip
 import shutil
+from datetime import datetime
 
-PYSIXDESK_ABSPATH=os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+#Gobal variables
+MES_TYPE = dict([[0,'Message'], [1, 'Warning'], [2, 'Error']])
```

Comment:
I would maybe slightly change the message levels into:
`MES_TYPE = dict([[0,'Debug'], [1, 'Warning'], [2,'Message'], [3, 'Error']])`
`Debug` would be the level for which the code is extremely verbose - such that almost every step could in principle give a message.
`Message` would simply be a regular message declaring that key steps are on-going. I propose that It has a higher level than warnings, since I think that one may want to run without warnings but still keeping an essential track of processing, what do you think?
Then, the default message level would still be 1.